modules/ROOT/pages/7.0.1-release-notes.adoc

```diff
@@ -78,13 +78,44 @@ The {productname} {release-version} release includes an accompanying release of
 
 This **Table of Contents** release includes the following fixes.
 
-==== Encode header title for literal display in table of contents.
+==== Encoding header of Table of Contents causing HTML source to be revealed on refresh.
+// #TINY-10682
+
+Previously in {productname}, the **Table of Contents** (ToC) header was encoded on refresh before being displayed in the editor. This resulted in the header being incorrectly shown as the underlying code literally, instead of being rendered as HTML.
```

Comment:
```suggestion
Previously in {productname}, the **Table of Contents** (ToC) header encoding caused special characters to be incorrectly displayed as code instead of their HTML representation.
```
The header is always encoded, but we are setting Table of Contents initially and there are no special characters so it shows up fine. Maybe?